src/pages/Beers/Beers.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import axios from "axios"
+import { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+function Beers() {
+  const [beers, setBeers] = useState([]);
+
+  const allBeers = async () => {
+    try {
+      const response = await axios.get(
+        "https://ih-beers-api2.herokuapp.com/beers"
+      );
+      setBeers(response.data);
+      console.log(response.data);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+  useEffect(() => {
+    allBeers();
+  }, []);
+
+  return (
+    <div>
+      <>
+        <Link to={`/`}>
+          <img className="Header"
+            src="https://user-images.githubusercontent.com/23629340/40707029-cb2fce12-63ef-11e8-939c-f673ff3b965d.png"
+            alt="Back"
+          />
+        </Link>
+      </>
+      {beers.map((beer) => (
```

Comment:
Here just be careful when rendering data from an api without any condition. This works but if the api takes some more time it will throw an error